app/views/propertyLinking/ownershipToProperty.scala.html

```diff
@@ -55,7 +55,7 @@
 
 @getEnrichedErrorLinks = @{
     toCustomDateFormErrors.toSet.map{ error: FormError =>
-        ErrorLink(href = Some(s"#${error.key}.day"),
+        ErrorLink(href = Some(s"#${error.key}.day" + s"#${error.key}" ),
```

Comment:
I think this should be either
```
s"#${error.key}.day"
```
or
```
s"#${error.key}"
```
but not
```
s"#${error.key}.day" + s"#${error.key}"
```

this code produces urls like: 
```
...property-links/ownership#fromDate.day#fromDate
```

I expect you want 
```
s"#${error.key}"
```